README.md

```diff
@@ -26,4 +27,39 @@ Then, load http://localhost:3000/ in your browser to access the app.
 ### Apps Security
 
 - [helmet.js](https://helmetjs.github.io/)
-- [express-session](https://www.npmjs.com/package/express-session)
\ No newline at end of file
+- [express-session](https://www.npmjs.com/package/express-session)
+
+
+## Axios
+
+Within this skeleton [axios](https://github.com/axios/axios) with [middleware-axios](https://github.com/krutoo/middleware-axios) (used a utility `../utils/axiosSetp.mjs` and can be extended with further middleware) is set up and ready to use out of the box.
+
+Below is an example of implementation of how to use the `axios_api` function, in other modules to make server/api calls:
+
+```mjs
+// routes/index.mjs
+import express from 'express';
+import axios_api from '../utils/axiosSetp.mjs';
+
+const router = express.Router();
+
+/* GET home page. */
+router.get('/', (req, res, next) => {
+  res.render('main/index', { title: 'Express' });
+});
+
+// Make an API call with `Axios` and `middleware-axios`
+// GET users from external API
+router.get('/users', async (req, res, next) => {
```

Comment:
This needs to be in the README.md

Reply:
@psweeting1 - this is the `README.md` ????

Reply:
WHOOPS! That's me. Github was telling me it was in app.mjs for some reason.